.github/workflows/daily.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will run integrations tests daily at midnight.
+
+name: Daily CI
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  test:
+    name: Daily Server Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        server: [eu-ger-1, eu-ger-2, eu-fin-1, us-or-1, us-or-2, us-pa-1, us-pa-2, us-va-1]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
```

Comment:
It would be nice to specify the version of Node we want thin run on.

e.g.:
```
  with:
    node-version: '12'
```
source: https://github.com/actions/setup-node

Reply:
For some reason I thought it would run the latest version. However, looking at the successful run in the OP it looks like it picked v10. Will change it.